pkg/module_manager/models/modules/basic.go

```diff
@@ -63,10 +65,13 @@ func NewBasicModule(name, path string, order uint32, staticValues utils.Values,
 		return nil, fmt.Errorf("new values storage: %w", err)
 	}
 
+	crdFilesPaths, _ := filepath.Glob(filepath.Join(path, "crds", "*.yaml"))
```

Comment:
could you please check that CRDs get isntalled even if they are located in subfolders, it seems some modules may have different structure of CRD directories like:
```
./crds/native/somecrd.yaml
```

Reply:
This function allows you to find files by mask only within one directory. If you need to use nested directories, you need to use filepath.Walk.

DONE